contrib/net/http/http_test.go

```diff
@@ -102,61 +103,80 @@ func TestWrapHandler200(t *testing.T) {
 }
 
 func TestAnalyticsSettings(t *testing.T) {
-	assertRate := func(t *testing.T, mt mocktracer.Tracer, rate interface{}, opts ...Option) {
-		mux := NewServeMux(opts...)
-		mux.HandleFunc("/200", handler200)
-		r := httptest.NewRequest("GET", "/200", nil)
-		w := httptest.NewRecorder()
-		mux.ServeHTTP(w, r)
-
-		spans := mt.FinishedSpans()
-		assert.Len(t, spans, 1)
-		s := spans[0]
-		assert.Equal(t, rate, s.Tag(ext.EventSampleRate))
+	asserts := []func(t *testing.T, mt mocktracer.Tracer, rate interface{}, opts ...Option){
+		func(t *testing.T, mt mocktracer.Tracer, rate interface{}, opts ...Option) {
+			mux := NewServeMux(opts...)
+			mux.HandleFunc("/200", handler200)
+			r := httptest.NewRequest("GET", "/200", nil)
+			w := httptest.NewRecorder()
+			mux.ServeHTTP(w, r)
+
+			spans := mt.FinishedSpans()
+			assert.Len(t, spans, 1)
+			s := spans[0]
+			assert.Equal(t, rate, s.Tag(ext.EventSampleRate))
+		},
+		func(t *testing.T, mt mocktracer.Tracer, rate interface{}, opts ...Option) {
+			f := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				message := "Hello \n"
+				w.Write([]byte(message))
+			})
+			handler := WrapHandler(f, "my-service", "my-resource", opts...)
+			r := httptest.NewRequest("GET", "/200", nil)
+			w := httptest.NewRecorder()
+			handler.ServeHTTP(w, r)
+
+			spans := mt.FinishedSpans()
+			assert.Len(t, spans, 1)
+			s := spans[0]
+			assert.Equal(t, rate, s.Tag(ext.EventSampleRate))
+		},
 	}
 
-	t.Run("defaults", func(t *testing.T) {
-		mt := mocktracer.Start()
-		defer mt.Stop()
+	for assert := range asserts {
```

Comment:
I think we are overusing and shadowing this name “assert”. It’s a test, maybe “test” is better? You could use the suggested map’s key string as the name

Reply:
You're right. There are too many things called "assert." I think a map with a string name is better.